local-cli/runAndroid/runAndroid.js

```diff
@@ -107,15 +108,29 @@ function buildAndRun(args, reject) {
       ? process.env.ANDROID_HOME + '/platform-tools/adb'
       : 'adb';
 
-    const adbArgs = [
-      'shell', 'am', 'start', '-n', packageName + '/.MainActivity'
-    ];
+    const devices = adb.getDevices();
+    
+    if(devices && devices.length > 0) {
+      devices.forEach((device) => {
 
-    console.log(chalk.bold(
-      'Starting the app (' + adbPath + ' ' + adbArgs.join(' ') + ')...'
-    ));
+        const adbArgs = ['-s', device, 'shell', 'am', 'start', '-n', packageName + '/.MainActivity'];
+
+        console.log(chalk.bold(
+          'Starting the app on ' + device + ' (' + adbPath + ' ' + adbArgs.join(' ') + ')...'
```

Comment:
Not sure if this file runs through Babel. If it does, use the ES2015 string interpolation syntax.